README.md

```diff
@@ -163,6 +163,15 @@ $ make install prefix=/path/to/install/location
 
 When running `make install` the default value of `prefix` is `$HOME/.local`.
 
+If you want `git help <skara tool>` (or the equivalent `man git-<skara tool>`
+to work, you must also add the `build/bin/man` directory to `$MANPATH`.
+For instance, run this from the Skara top directory to add this to your
+`.bashrc` file:
+
+```
```

Comment:
```suggestion
```bash
```

GitHub Flavored Markdown (and regular Markdown) support an [info string](https://github.github.com/gfm/#info-string) after the opening code fence symbols (```). This info string is commonly used to describe the programming language used in a code block (`bash` in this example). This enables for example syntax highlighting etc.